tests/test_requests.py

```diff
@@ -401,6 +401,115 @@ def test_content_type(app):
     assert response.text == "application/json"
 
 
+def test_standard_forwarded(app):
+    @app.route("/")
+    async def handler(request):
+        return json(request.forwarded)
+
+    # Without FORWARDED_SECRET, X-headers should be respected
+    app.config.PROXIES_COUNT = -1
+    headers = {
+        "Forwarded": (
+            'for=1.1.1.1, for=injected;host="'
```

Comment:
Each proxy will append its entry to the end, comma separated or as an additional header row. If proxies are not careful in examining the prior entries for correct syntax (and you can be sure that many of them will just blindly append to anything), malformed results like this may occur. We mitigate such problems by:
- Parsing right-to-left, starting with the more trustworthy most-local proxies.
- Parse errors clear any keys found and skip to the next comma (so that malicious for=injected is not combined with secret=mySecret added by the true proxy and accidentally accepted).